content/docs/1_guide/2_read-me/guide.txt

```diff
@@ -0,0 +1,250 @@
+Title: Read me first
+
+----
+
+Description: A quick overview on the different parts of the documentation
+
+----
+
+Intro: Guide, Reference, Cookbook, Search, where to find what and how to best use them
+
+----
+
+
+Text:
+
+Every user comes to a documentation with different expectations, background and prior knowledge. Ideally, Kirby or any other software/product would be self-explanatory, but that's a dream. It is equally a dream to make everyone happy, and consequently some of you will love this documentation and some of you will hate it, as our last survey clearly showed. But that's no excuse, and we will strive to improve it constantly.
+
+## Parts of the documentation
+
+This documentation has <del>three</del> four parts:
+
+### 1. Guide:
+That's where you currently are and where you should start when you are new to Kirby to get an overview on the system. The Guide is divided into topics, and is the place where beginners should start.
```

Comment:
I think the second sentence is duplicated because the topics are mentioned below and the "beginners should start" part is mentioned directly before.

```suggestion
That's where you currently are and where you should start when you are new to Kirby to get an overview on the system.
```